.gitmodules

```
@@ -1,4 +1,4 @@
 [submodule "fin"]
     path = fin
-    url = https://github.com/ROCmSoftwarePlatform/MIFin.git
+    url = https://github.com/ROCm/MIFin.git
```

Because we're all in the same organization now, you may define this as a relative path. For example,
```suggestion
    url = .../MIFin
```